recipes/NNPOps/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "nnpops" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/openmm/NNPOps/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 64eb48e4a2a642f5e951fe368c46bd41402ce68a89f2f56189728b4f551cbe9d
+
+build:
+  skip: true  # [(not linux) or (cuda_compiler_version in (undefined, 'None'))]
+  number: 0
+  rpaths:
+    - lib/
+    - {{ SP_DIR }}/torch/lib
+  missing_dso_whitelist:
+    - '*/libcuda.*'  # [linux64 or ppc64le]
+  string: cuda{{ cuda_compiler_version | replace('.', '') }}py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [cuda_compiler_version != "None"]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}  # [cuda_compiler_version not in (undefined, 'None')]
+    - sysroot_linux-64  2.17  # [linux64]
+    - make
+    - cmake
+  host:
+    - python 
+    - pytorch 1.10.0
+    - pytorch =*=*cuda*
+    - torchani
+    - mdtraj
```

Comment:
`mdtraj` and `pytest` are only needed for testing.

Reply:
I removed `mdtraj` as a run time dependency, but kept it (and `pytest`) as host dependencies since we do run the tests when we build the package 
https://github.com/conda-forge/staged-recipes/pull/16257/files#diff-b999cf82f5d4187d1689de4805691ea3d6da5b925c3e425c67a2cc39a34f7f38R53

Reply:
I would reconsider this in the feedstock. Running tests as part of the build doesn't cover all cases. For example, conda-build will make sure the test env paths are different from the build time paths so we ensure libraries can be found across different installation paths. Otherwise we never know if binary prefix relocation works.

If you have tested this locally and you could run the tests, that's ok, but this is not as ideal as it looks like right now.

Reply:
So is it better to run the tests during the "test" step then? I have tested this locally, and it has been working for other people as well. So maybe we could address this post merge?